modules/quota-establishment-api.adoc

```diff
@@ -80,7 +81,14 @@ $ curl -k -X PUT -H "Authorization: Bearer <token>" -H 'Content-Type: applicatio
 
 To see the storage consumed, push various images to the organization.
 
-=== Pushing ubuntu:18.04
```

Comment:
I am going to leave this as a single procedure, rather than breaking down into multiple micro procedures.

I want to keep the UI and API document in sync